extensions/amp-smartlinks/0.1/linkmate.js

```diff
@@ -163,12 +163,30 @@ export class Linkmate {
    * @private
    */
   getEditInfo_() {
+    let currentUrl = this.ampDoc_.getUrl();
+    if (currentUrl.includes('www-')) {
```

Comment:
Would a regex for `https://www.` where the s is optional be more thorough?

Reply:
Kind of tough to say considering we've only seen this specific example. Removing the conditional would probably be more thorough, just extracting the base of every url to be sure.